src/initialize_dae.jl

```diff
@@ -147,10 +148,11 @@ function _initialize_dae!(integrator, prob::ODEProblem,
 	update_coefficients!(M,u0,p,t)
 	algebraic_vars = [all(iszero,x) for x in eachcol(M)]
 	algebraic_eqs  = [all(iszero,x) for x in eachrow(M)]
+  (iszero(algebraic_vars) || iszero(algebraic_eqs)) && return
```

Comment:
why?

Reply:
`iszero(algebraic_vars)` or `iszero(algebraic_eqs)` means the nlsys doesn't have any input or output.

Reply:
ahh